OrbitCore/SamplingProfiler.h

```diff
@@ -92,9 +94,12 @@ class SamplingProfiler {
     return &m_Callstacks;
   }
 
-  const std::unordered_map<CallstackID, std::shared_ptr<CallStack>>&
-  GetUniqueCallstacks() const {
-    return m_UniqueCallstacks;
+  template <typename F>
```

Comment:
Why the template instead of passing a `std::function<void(const CallStack&)>`?

Reply:
habbit from before std::function. fixed.